src/typings/tacoLiveReload.d.ts

```diff
@@ -53,6 +53,7 @@ declare module TacoLiveReload {
     export interface LiveReloadOptions {
         ignore: string,
         ghostMode: boolean,
+        tunnel: boolean,
```

Comment:
I've added a description for tunnel. The ghostMode option is the internal name for deviceSync. We should document the ignore option, I'll get going on that.